src/amo/components/UserProfileEdit/index.js

```diff
@@ -525,11 +527,11 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                     dangerouslySetInnerHTML={sanitizeHTML(
                       i18n.sprintf(
                         i18n.gettext(`You can change your email address on
-                            Firefox Accounts. %(startLink)sNeed help?%(endLink)s`),
+                            Firefox Accounts. %(linkStart)sNeed help?%(linkEnd)s`),
```

Comment:
All this does is rename the variables, right? If so, I don't think it's worth making this change. Once it lands, all previous localizations will become invalid and each localizer will have to start over again. They will be able to re-use their old localization but it will be a lot of work for them.